nsv13/code/datums/missions.dm

```diff
@@ -67,6 +68,10 @@ NOTE:
     return FALSE // If the mission is not active, it can not be completed
 
 /datum/nsv_mission/proc/payout()
+  var/ownerFaction = owner.faction
+  for(var/datum/faction/NT in SSstar_system.factions)
+    if( ownerFaction == lowertext( NT.name ) )
```

Comment:
This would mean that on PVP, syndicates completing missions would benefit Nanotrasen. More stringent checking's required here, why not look at the valid_factions list for inspo?